path_static_roles.go

```diff
@@ -20,10 +21,24 @@ const (
 	staticRolePath = "static-role/"
 )
 
+// GenericNameWithForwardSlashRegex is a regex which requires a role name. The role name can
+// include any number of alphanumberic characters separated by forward slashes.
+func GenericNameWithForwardSlashRegex(name string) string {
```

Comment:
It would be great if we could have a small unit test validating this regex does what we expect. I personally am not great with regexes and usually am unable to tell if regexes run as expected 😅 

Disregard if this is not a concern! :)

Reply:
I planned to add tests for this but it slipped my mind. I will add some. Eventually, I plan to move this to the Vault sdk but for now we will embed them in the plugin